ui/src/main/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtils.java

```diff
@@ -125,4 +131,52 @@ public static String getTransferMethodName(@NonNull final Context context,
 
         return title;
     }
+
+    /**
+     * Gets Transfer method identifier from the {@link HyperwalletTransferMethod} field
+     * by a {@link TransferMethodType}.
+     *
+     * @param context        Context
+     * @param transferMethod HyperwalletTransferMethod
+     * @param type           TransferMethodType
+     */
+    public static String getTransferMethodDetail(@NonNull Context context,
+            @NonNull final HyperwalletTransferMethod transferMethod,
+            @TransferMethodType final String type) {
```

Comment:
add `@NonNull` if applicable `@Nullable` if not, but I see you checked for null after this statements so if we will know 100% probably we will have to annotate its easier to read

Reply:
Added